OWDTestToolkit/apps/dialer.py

```diff
@@ -412,7 +412,8 @@ def hangUp(self):
             self.UTILS.iframe.switchToFrame(*DOM.Dialer.frame_locator_calling)
             self.marionette.find_element(*DOM.Dialer.hangup_bar_locator).tap()
             self.apps.switch_to_displayed_app()
-        except:
+        except Exception, e:
```

Comment:
Syntax:

    except Exception as e:

is preferred in modern Python versus the comma-based old one.